static/src/uu/index.scss

```diff
@@ -14,7 +14,7 @@
     }
 
     /* File browser rows */
-    .browse, .deposit-delete, .page-link, .system-metadata-items, .name {
+    .browse, .deposit-delete, .page-link, .system-metadata-items, .name, .dlg-browse {

```

Comment:
This makes the look of the file browser when copying or moving files more unified with the general dark mode.